content/admin/administration-panel/Versions.adoc

```diff
@@ -0,0 +1,33 @@
+---
+Title: Version Control(Release)
+Weight: 80
+---
+
+:experimental:
+
+:imagesdir: ./../../../images/en/user
+
+= Version Control
+
+In this section you can manage the list of versions of your products. The list of versions is displayed in the drop-down list when describing a new
+link:../../../user/core-modules/bugs[bugs].
```

Comment:
the label should be just `bug`, though the link is to `bugs`, so that it fits the sentence